src/Node/ExampleNode.php

```diff
@@ -198,14 +198,16 @@ protected function replaceArgumentsTokens(array $arguments)
      */
     protected function replaceTableArgumentTokens(TableNode $argument)
     {
-        $table = $argument->getTable();
-        foreach ($table as $line => $row) {
-            foreach (array_keys($row) as $col) {
-                $table[$line][$col] = $this->replaceTextTokens($table[$line][$col]);
```

Comment:
I rewrote this by building a new array from scratch instead of copying the array (due to copy on write) and then overriding each value in place, because phpstan is not smart enough to identify that the `$col` key is an existing key of the list here, and so it preserves the fact that the array is a list.